### PR TITLE
Feature/comment show on wave

### PR DESCRIPTION
**Pull request by Naoyakaoru**
1. add comment space below soundwave
2. get duration through shrine
3. can record timepoint in show
4. can append avatar when focus on comment bar
5. modify body position fixed to body overflow hidden to avoid view error
6. list todos

### CODE REVIEW

**Review by tungchinlee — APPROVED** [review]

**Review by leegocode — APPROVED** [review]

File: app/views/library/sets.html.erb [hunk 1]
```diff
@@ -88,4 +90,5 @@
     </div>
   </div>
 >>>>>>> master
+>>>>>>> master
```
Comment (boypie510) [inline app/views/library/sets.html.erb:93]: 這個沒有解完衝突？

File: app/views/songs/index.html.erb [hunk 2]
```diff
@@ -17,7 +17,11 @@
             <li><%= tag.name %></li>
           <% end %>
         <% end %>
+<<<<<<< HEAD
+        <%= content_tag(:li, 'play ▶', class:['getURL song-place'], data: {id: song.id} ) %>
+=======
         <%= content_tag(:li, 'play ▶', class:'getURL', data: {id: song.id} ) %>
+>>>>>>> master
```
Comment (boypie510) [inline app/views/songs/index.html.erb:24]: 這個看起來也是還沒解完衝突？

**Review by TeKaiPeng — APPROVED** [review]

File: app/views/library/sets.html.erb (hunk 1 above)
Comment (Naoyakaoru) [reply app/views/library/sets.html.erb]: fixed in 34abc23

File: app/views/songs/index.html.erb (hunk 2 above)
Comment (Naoyakaoru) [reply app/views/songs/index.html.erb]: fixed in 34abc23

**Review by boypie510 — APPROVED** [review]

**Pull request merged**